src/Common/Game/IW5/IW5_Assets.h

```diff
@@ -4380,6 +4380,19 @@ namespace IW5
         LBUPDATE_TYPE_COUNT = 0x3
     };
 
+    enum LbTrackType
+    {
+        TRK_ALLTIME = 0x0,
+        TRK_WEEKLY = 0x1,
+        TRK_MONTHLY = 0x2,
+        TRK_PRESTIGE_ALLTIME = 0x3,
+        TRK_PRESTIGE_WEEKLY = 0x4,
+        TRK_PRESTIGE_MONTHLY = 0x5,
+        TRK_DAILY = 0x6,
+        TRK_PRESTIGE_DAILY = 0x7,
+        TRK_COUNT = 0x8,
```

Comment:
i try to not hardcode the count value but let the compiler select it itself, so adding an entry only adds a line but not change the count property and prevents accidents in not increasing it.